_posts/2019-07-29-vim-basics.markdown

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: vim Basics
```

Comment:
@bronzdoc Maybe you can change the title to be Vim Guide for Beginners, or something like that?

Reply:
Sounds good 👍